meta-resin-supervisor/recipes-devtools/nodejs/nodejs/0001-backport-dns-add-verbatim-option-to-dns.lookup.patch

```diff
@@ -0,0 +1,219 @@
+From 48165469acafb2e82adbf7798084b08e39849dd5 Mon Sep 17 00:00:00 2001
+From: Ben Noordhuis <info@bnoordhuis.nl>
+Date: Thu, 10 Aug 2017 11:19:13 +0200
+Subject: [PATCH] (backport) dns: add `verbatim` option to dns.lookup()
+
+When true, results from the DNS resolver are passed on as-is, without
+the reshuffling that Node.js otherwise does that puts IPv4 addresses
+before IPv6 addresses.
+
+PR-URL: https://github.com/nodejs/node/pull/14731
+Ref: https://github.com/nodejs/node/issues/6307
+Reviewed-By: Refael Ackermann <refack@gmail.com>
+Reviewed-By: Colin Ihrig <cjihrig@gmail.com>
+Reviewed-By: James M Snell <jasnell@gmail.com>
```

Comment:
NIT - It would be nice to have a Upstream status tag as well: http://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines

Reply:
Good point, done